### ISSUE TITLE
Still having drawing bug where masks wont actually be created once drawn

### ISSUE THREAD

**Issue by mayishazn** [MouseLand/cellpose#1051]
I created a new environment and did a fresh install. The red circle does disappear once I close the loop but it doesn't turn into a mask. I've drawn four here and they're just stuck with the purple dotted lines. There is no drawing related output in the terminal either.

![image](https://github.com/user-attachments/assets/3530111d-e129-402d-9553-8c32c001779a)

_Originally posted by @mayishazn in https://github.com/MouseLand/cellpose/issues/961#issuecomment-2456358482_

Also sorry for the repeat issue. It would just be highly convenient if I could fix masks on cellpose 3 directly.

**Comment by carsen-stringer** [comment]
oh no, I was hoping I had found the bug! Does this happen when you start from scratch: can you delete your _seg.npy file and start labeling again and confirm it happens? can you also please export your environment `conda env export > cp.yml` and share it here? I will try to reproduce. how big is your image?

**Comment by mayishazn** [comment]
name: freshcellpose3
channels:
  - pytorch
  - nvidia
  - defaults
dependencies:
  - blas=1.0=mkl
  - bzip2=1.0.8=h2bbff1b_6
  - ca-certificates=2024.9.24=haa95532_0
  - console_shortcut=0.1.1=haa95532_6
  - cuda=11.6.1=0
  - cuda-cccl=11.6.55=0
  - cuda-command-line-tools=11.6.2=0
  - cuda-compiler=11.6.2=0
  - cuda-cudart=11.6.55=0
  - cuda-cudart-dev=11.6.55=0
  - cuda-cuobjdump=11.6.124=0
  - cuda-cupti=11.6.124=0
  - cuda-cuxxfilt=11.6.124=0
  - cuda-libraries=11.6.1=0
  - cuda-libraries-dev=11.6.1=0
  - cuda-memcheck=11.8.86=0
  - cuda-nsight-compute=12.4.1=0
  - cuda-nvcc=11.6.124=0
  - cuda-nvdisasm=12.6.77=0
  - cuda-nvml-dev=11.6.55=0
  - cuda-nvprof=12.6.80=0
  - cuda-nvprune=11.6.124=0
  - cuda-nvrtc=11.6.124=0
  - cuda-nvrtc-dev=11.6.124=0
  - cuda-nvtx=11.6.124=0
  - cuda-nvvp=12.6.80=0
  - cuda-runtime=11.6.1=0
  - cuda-sanitizer-api=12.6.77=0
  - cuda-toolkit=11.6.1=0
  - cuda-tools=11.6.1=0
  - cuda-version=12.6=3
  - cuda-visual-tools=11.6.1=0
  - expat=2.6.3=h5da7b33_0
  - fontconfig=2.14.1=hb33846d_3
  - freetype=2.12.1=ha860e81_0
  - gmpy2=2.1.2=py311h7f96b67_0
  - intel-openmp=2023.1.0=h59b6b97_46320
  - jinja2=3.1.4=py311haa95532_0
  - libcublas=11.9.2.110=0
  - libcublas-dev=11.9.2.110=0
  - libcufft=10.7.1.112=0
  - libcufft-dev=10.7.1.112=0
  - libcurand=10.3.7.77=0
  - libcurand-dev=10.3.7.77=0
  - libcusolver=11.3.4.124=0
  - libcusolver-dev=11.3.4.124=0
  - libcusparse=11.7.2.124=0
  - libcusparse-dev=11.7.2.124=0
  - libffi=3.4.4=hd77b12b_1
  - libglib=2.78.4=ha17d25a_0
  - libiconv=1.16=h2bbff1b_3
  - libnpp=11.6.3.124=0
  - libnpp-dev=11.6.3.124=0
  - libnvjpeg=11.6.2.124=0
  - libnvjpeg-dev=11.6.2.124=0
  - libpng=1.6.39=h8cc25b3_0
  - libuv=1.48.0=h827c3e9_0
  - libxml2=2.13.1=h24da03e_2
  - mkl=2023.1.0=h6b88ed4_46358
  - mpc=1.1.0=h7edee0f_1
  - mpfr=4.0.2=h62dcd97_1
  - mpir=3.0.0=hec2e145_1
  - mpmath=1.3.0=py311haa95532_0
  - nsight-compute=2024.3.2.3=0
  - openssl=3.0.15=h827c3e9_0
  - pcre2=10.42=h0ff8eda_1
  - pip=24.2=py311haa95532_0
  - python=3.11.10=h4607a30_0
  - pytorch-cuda=11.6=h867d48c_1
  - pytorch-mutex=1.0=cpu
  - pyyaml=6.0.2=py311h827c3e9_0
  - setuptools=75.1.0=py311haa95532_0
  - sqlite=3.45.3=h2bbff1b_0
  - tbb=2021.8.0=h59b6b97_0
  - tk=8.6.14=h0416ee5_0
  - typing_extensions=4.11.0=py311haa95532_0
  - tzdata=2024b=h04d1e81_0
  - vc=14.40=h2eaa2aa_1
  - vs2015_runtime=14.40.33807=h98bb1dd_1
  - wheel=0.44.0=py311haa95532_0
  - xz=5.4.6=h8cc25b3_1
  - yaml=0.2.5=he774522_0
  - zlib=1.2.13=h8cc25b3_1
  - pip:
      - cellpose==3.1.1.dev1+g9398cac
      - colorama==0.4.6
      - fastremap==1.15.0
      - filelock==3.16.1
      - fsspec==2024.10.0
      - imagecodecs==2024.9.22
      - llvmlite==0.43.0
      - markupsafe==3.0.2
      - natsort==8.4.0
      - networkx==3.4.2
      - numba==0.60.0
      - numpy==2.0.2
      - opencv-python-headless==4.10.0.84
      - packaging==24.1
      - pillow==10.2.0
      - pygments==2.18.0
      - pyqt6==6.7.1
      - pyqt6-qt6==6.7.3
      - pyqt6-sip==13.8.0
      - pyqtgraph==0.13.7
      - qtpy==2.4.2
      - roifile==2024.9.15
      - scipy==1.14.1
      - superqt==0.6.7
      - sympy==1.13.1
      - tifffile==2024.9.20
      - torch==2.5.1+cu118
      - torchaudio==2.5.1+cu118
      - torchvision==0.20.1+cu118
      - tqdm==4.66.6
      - typing-extensions==4.12.2
prefix: C:\Users\mayis\anaconda3\envs\freshcellpose3

**Comment by mayishazn** [comment]
These are 2048x2048 images and some of them are zstacks with 10 slices. 
It seems to happen regardless of if I'm starting from scratch or fixing masks that cyto3 found. 
![image](https://github.com/user-attachments/assets/35eb3d91-5612-4cb1-9928-e6bba7ea254c)

**Comment by mayishazn** [comment]
idk if the image type matters? These are tiffs

**Comment by carsen-stringer** [comment]
For the stacks are you opening them with --Zstack? In that case single stroke is off

**Comment by mayishazn** [comment]
Yes I am opening them with --Zstack. So how do I edit them then?

**Comment by carsen-stringer** [comment]
Ahha that's it, you have to press enter to complete the mask, or you can turn on single stroke

**Comment by mayishazn** [comment]
![image](https://github.com/user-attachments/assets/89366815-f53e-4756-bc10-302376e3ad50)
This works for a few masks and then it gets stuck on draw mode again.

**Comment by carsen-stringer** [comment]
I think @mrariden tracked this down!!! Please try the latest cellpose version
